config/ankhconfig.yaml

```diff
@@ -1,9 +1,51 @@
-current-context: minikube
+# current_context defines which context you currently have "selected". It's
+# common for this value to change frequently as you move around to different
+# clusters.
+current_context: minikube_local
+
+# supported_environments define a list of your supported environments. Used for
+# validation of ankh files. It's best practice to keep this value in sync with
+# the rest of your teams so there isn't drift.
+supported_environments:
+  - dev
+  - production
+
+# supported_profiles define a list of your supported profiles. Used for
+# validation of ankh files. Profiles are typically used for values that relate
+# to how beefy your app needs to be. If you're running a production cluster that
+# customers can hit, you'll probably want to support a value like "natural".
+# Alternatively if you want your app to run on a resource strapped local
+# cluster, you might consider also supporting something like "constrained". It's
+# best practice to keep this value in sync with the rest of your teams so there
+# isn't drift.
+supported_profiles:
```

Comment:
I'm cool with just using `profiles` instead of `resource_profiles` unless someone _really_ wants the other one.

Reply:
use 'supported-resource-profiles' for consistency with the existing language